.evergreen/config.yml

```diff
@@ -1391,7 +1395,7 @@ buildvariants:
       - name: ".test !.enterprise-auth !.snappy !.zstd"
 
   - matrix_name: "tests-42-lts-zlib-zstd-support"
```

Comment:
Seems like we should update the matrix name to reflect that it's no longer only 4.2

Reply:
Changed to `tsts-42-plus...`

Reply:
Better to let the original commenter resolve the conversation.  Easier to do the follow-up review that way.